mypy/stubgen.py

```diff
@@ -1043,8 +1275,6 @@ def generate_stub_from_ast(mod: StubSource,
     if subdir and not os.path.isdir(subdir):
         os.makedirs(subdir)
     with open(target, 'w') as file:
-        if add_header:
-            write_header(file, mod.module, pyversion=pyversion)
```

Comment:
Maybe we can keep this as opt-in, but make it more informative, like add a creation date, mypy version, etc.?

Reply:
TBH, I don't think that the header is useful, since it will likely get stale once the stub is updated.